.github/workflows/scripts/push-images.sh

```diff
@@ -48,3 +48,6 @@ echo "Executing: docker tag $LOCALIMG $REMOTEIMG"
 docker tag "$LOCALIMG" "$REMOTEIMG"
 echo "Executing: docker push $REMOTEIMG"
 docker push "$REMOTEIMG"
+
+image_digest="$(docker inspect "${REMOTEIMG}" --format '{{ index .RepoDigests 0 }}' | awk -F '@' '{ print $2 }')"
```

Comment:
This will have to change if #70 is merged first. If this one is merged first I will rebase #70 to adjust it.